src/output.css

```diff
@@ -914,6 +914,10 @@ body {
   height: 100vh;
 }
 
+.h-5\/6 {
+  height: 83.333333%;
+}
+
 .max-h-96 {
```

Comment:
`output.css` Don't we have any `scss` files with some structure? 
Everything is in one, two files? index.css and output.css?

Reply:
I believe it's generated 🤞 
But if not - we should definitely split those large file(s). :)
If not, @Nazarii-4chain - could you explain when the `output.css` and similar are generated?

Reply:
We don't have `scss` files. Instead we are using `tailwind`. `output.css` generated by tailwind.
Output.css regenerate each time when styles changes in the project.

Reply:
Hmm so this should not be ignored by git if its generated? 

I have used tailwind and there was no autogenerated file. We just import tailwind to the project and uses tailwind classes as usual. Like any other css class 🤔

Reply:
It should work like you described. But it does not update styles in real time.
https://tailwindcss.com/docs/content-configuration#it-just-isnt-working-properly

Reply:
@Nazarii-4chain I've fixed this issue. This was not a problem with tailwind. 
PR: https://github.com/bitcoin-sv/spv-wallet-admin/pull/971